api/http/handler/file.go

```diff
@@ -35,6 +35,9 @@ func isHTML(acceptContent []string) bool {
 func (handler *FileHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if !isHTML(r.Header["Accept"]) {
 		w.Header().Set("Cache-Control", "max-age=31536000")
+		w.Header().Set("X-XSS-Protection", "\"1; mode=block\"")
```

Comment:
If the headers are added in this `if` block, they'll only be added when sending HTTP requests to get HTML files. Should these headers be added for other requests as well ? (e.g Images, Javascript files...) ?